src/qonlinetts.cpp

```diff
@@ -161,6 +182,111 @@ QOnlineTts::Voice QOnlineTts::voice(const QString &voiceCode)
     return s_voiceCodes.key(voiceCode, NoVoice);
 }
 
+QPair<QOnlineTranslator::Language, QLocale::Country> QOnlineTts::region(const QString &regionCode)
+{
+    return s_regionCodes.key(regionCode, {QOnlineTranslator::NoLanguage, QLocale::AnyCountry});
+}
+
+QString QOnlineTts::regionName(QOnlineTranslator::Language language, QLocale::Country region)
+{
+    switch (language) {
+    case QOnlineTranslator::Bengali:
+        switch (region) {
+        case QLocale::Bangladesh:
+            return tr("Bangla (Bangladesh)");
+        case QLocale::India:
+            return tr("Bangla (India)");
+        default:
+            return tr("Default region");
```

Comment:
We can simplify it. Let's remove all duplicated `default` and add `return tr("Default region");` at the end of the function.

Reply:
9 warnings are generated in Qt Creator this way. Is it proper add this piece of macro?
`#pragma GCC diagnostic ignored "-Wswitch"`

Reply:
Right, it complains about unhandled things in enum...
I have a better idea. Let's remove this function :) We already have `languageName` for languages and Qt provides a way to get country name. We will just concatenate them in Crow Translate. Less code - better :)

Reply:
I was thinking of that, actually. But things are a little more complicated when it comes to the Chinese language(s).
To describe a TTS output voice for Chinese, both the dialect (e.g. Mandarin, Hakka) and the country must be given. "Chinese, Mandarin (China)" cannot be replaced by "Simplified Chinese (China)". The "simplified" adjective only describes the writing system. It is not logical to call a spoken output "simplified" or "traditional". And even a given dialect may be spoken in different ways in different countries. Currently Google only supports "Chinese, Mandarin (China)". But their paid TTS now supports Chinese, Mandarin (Taiwan) and Chinese, Yue (China). One day more maybe supported in the free Translate TTS API.

Reply:
Okay, let's keep it.
Here is what we can do to eliminate the warning: bring `default` back, but put `break;`. This will remove the warning and allow as to use a single return with default.